templates/core/features/index.html

```diff
@@ -4,314 +4,276 @@
+          <p class="p-equal-height-row__item">Over-the-air (OTA) updates guarantee secure and stable software in any location. You decide which updates are signed, certified and delivered to your devices. Plus, Ubuntu Core keeps the last working boot so you always have a safety net.</p>
+        </div>
+        <div class="p-equal-height-row__col">
+          <h3 class="p-heading--5 p-equal-height-row__item">
+            <a href="/internet-of-things/management">Fleet management&nbsp;&rsaquo;</a>
+          </h3>
+          <p class="p-equal-height-row__item">Landscape delivers comprehensive management capabilities across the full scope of your device fleets. Stay in control with features including canary releases, remote device remodelling and system monitoring.</p>
+        </div>
+        <div class="p-equal-height-row__col">
+          <h3 class="p-heading--5 p-equal-height-row__item">
+            <a href="/core/features/secure-boot">Validations sets&nbsp;&rsaquo;</a>
+          </h3>
+          <p class="p-equal-height-row__item">Validation sets guarantee the installation of required applications with fine-grained permissions. What’s more, you can update applications consistently and simultaneously, improving out-of-the-box experience for end-users.</p>
+        </div>
+      </div>
+      <div>
+        
+      </div>
     </div>
   </div>
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h2>Secure boot</h2>
-      <p>Ubuntu Core 20 authenticates the boot process by default. Authentication is based on the verification of digital signatures. This means:</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Each component in the boot sequence cryptographically validates the authenticity of the subsequent component in the boot sequence.</li>
-        <li class="p-list__item is-ticked">Every component is measured, before it is loaded in the runtime memory space</li>
-        <li class="p-list__item is-ticked">If an improper or unsigned component is detected, the boot process is stopped</li>
-        <li class="p-list__item is-ticked">Supports for both hardware and software Root of Trust</li>
-      </ul>
-      <p>
-        <a href="/core/features/secure-boot">Learn more about secure boot&nbsp;&rsaquo;</a>
-      </p>
-    </div>
-    <div class="col-4 u-align--center u-vertically-center u-hide--medium u-hide--small">
+  <div class="row">
+    <div class="col-9 col-start-large-4 col-medium-6">
+      <div>
         {{ image (
-          url="https://assets.ubuntu.com/v1/6ed19d21-UC20_Secure_boot.svg",
-          alt="",
-          width="160",
-          height="160",
+         url="https://assets.ubuntu.com/v1/4769d9d3-updates.jpg",
+          alt="Drone grounded close to what seems to be a corn field",
+          width="916",
+          height="392",
           hi_def=True,
-          loading="lazy"
+          loading="auto"
```

Comment:
Why auto? We usually reserve auto loading for images that are visible in the vp on load